templates/config.yaml

```diff
@@ -565,6 +565,11 @@ templates:
           <<: *console_generic_component
           path: tests
           since: "1.11"
+      - to: ../prow/jobs/console/core/core.tmp.yaml
+        values:
+          <<: *console_generic_component
```

Comment:
Probably different pre-component should be used. Core is not dependent on components / shared